### DISCUSSION TITLE
Hindi Translation

### DISCUSSION THREAD

**Discussion by thetejasmagade** [vuejs-translations/guidelines#38] [Show and tell]
I just seen that someone is opened a discussion for hindi translation but he is not working on it. So, because of that I'm starting a new discussion for "Hindi Translation"

Anyone wants to join just enter into the discussion 🤠

**Comment by Akshit1025** [comment]
I am also interested

**Comment by Akshit1025** [comment]
> I just seen that someone is opened a discussion for hindi translation but he is not working on it. So, because of that I'm starting a new discussion for "Hindi Translation"
> 
> Anyone wants to join just enter into the discussion 🤠

Hey, I have already started the translation of the documentation at htt[s://github.com/Akshit1025/hn.vuejs.org

Please come to that repository and you can contribute there as I am already ahead of you in translation
